### DISCUSSION TITLE
errno = 48

### DISCUSSION THREAD

**Discussion by klarqs** [serverpod/serverpod#1114] [Q&A] [answered]
I am a Flutter developer from Lagos, Nigeria. I recently stumbled upon Serverpod - A great serverside option for Flutter.
Currently, I'm encountering a websocket issue - SocketException: Failed to create server socket (OS Error: Address already in use, errno = 48)

<img width="1792" alt="Screenshot 2023-07-11 at 12 13 31 AM" src="https://github.com/serverpod/serverpod/assets/63057921/8d6490c3-51de-4302-9014-4d1044cd5497">

Kindly help, thanks in adv...

**Comment by jayjah** [comment, marked as answer]
@kola-rt Your serverpod server uses the port `8080`, but this port is already in use by some other application. So you check which software uses that port and close that application or just choose another one for your serverpod server.

**Reply by vlidholt** [reply]
You can also configure which port you want Serverpod to use in the `config/development.yaml` file.
